tests/test_models.py

```diff
@@ -550,6 +666,18 @@ def test_draft_status_for_brief_response_with_no_submitted_at(self):
         brief_response = BriefResponse(created_at=datetime.utcnow())
         assert brief_response.status == 'draft'
 
+    def test_awarded_status_for_brief_response_has_been_awarded(self):
+        with self.app.app_context():
+            brief = Brief.query.get(self.brief_id)
+            brief_response = BriefResponse(
+                brief=brief,
+                submitted_at=datetime.utcnow(),
+                award_details={'confirmed': 'details'},
+                awarded_at=datetime(2016, 1, 1)
+            )
+
```

Comment:
To check for the sql `hybrid_property.expression` case we can add, commit and reload the brief_response before checking the status